packages/next-swc/crates/next-core/src/bootstrap.rs

```diff
@@ -240,6 +243,24 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_replace_percent_40() {
+        assert_eq!(
+            normalize_app_page_to_pathname("/replace%40with_underscore"),
+            "/replace%40with_underscore"
+        );
+        assert_eq!(
+            normalize_app_page_to_pathname("/%40replace%40with_underscore"),
+            "/_replace%40with_underscore"
```

Comment:
Shouldn't this be `"/@replace%40with_underscore"`? The test path should probably also contain `with_at` instead of `with_underwcore`.

Reply:
Yeah you are right, I don't know why I just blindly copied and pasted the tests here especially when these things are so clear 🤦‍♂️. Will fix now.